test/test_evaluation/test_train_evaluator.py

```diff
@@ -3346,3 +3506,47 @@ def test_eval_partial_cv(self):
             self.assertEqual(len(rval), 1)
             self.assertAlmostEqual(rval[0]["loss"], results[fold])
             self.assertEqual(rval[0]["status"], StatusType.SUCCESS)
+
+    def test_eval_partial_cv_multi_objective(self):
+        metrics = {
+            accuracy: [
+                0.050000000000000044,
+                0.0,
+                0.09999999999999998,
+                0.09999999999999998,
+                0.050000000000000044,
+            ],
+            balanced_accuracy: [
+                0.04761904761904756,
+                0.0,
+                0.10317460317460314,
+                0.11111111111111116,
+                0.05555555555555547,
+            ],
+        }
+
+        for fold in range(5):
+            instance = json.dumps({"task_id": "data", "fold": fold})
```

Comment:
[Q]
I don't think I've ever seen the need for this in the code I've seen, is this `instance` passed directly to `SMAC` which expects `json`?

Reply:
Yes, it is passed to SMAC, and SMAC then passes it to the target function. SMAC requires a string. To make parsing that string easier later on, we encode it to a json that we can easily decode inside the target function.